index.rst

```diff
@@ -85,12 +103,12 @@ This includes:
 - Procedures to enter telescope spaces during operations (e.g. AuxTel)
 - Emergency meeting locations etc.
 
+.. _Top-Project-Documentation-Information:
 
 Project Documentation Information
 =================================
 
-- Describes information regarding this documentation tree, including how to contribute.
-
+Describes information regarding this documentation tree, including how to contribute.
 
 .. toctree::
     :maxdepth: 2
```

Comment:
if you look at the rendered toc, then you see the templates are shown in the TOC and it's a bit confusing without context. I wonder if it's best to create a "Templates directory" or something ? 
Or maybe just exclude them from the TOC?

Reply:
Created a directory "project/Templates" which contains it's own index file, the .rst template files (moved the tutorial-template here) and the Template-Folder (added to exclude_patterns in the conf.py). The TOC on the main page includes a "Template Folder Index" link, but doesn't contain the templates themselves. The templates are available on the project page and, obviously, the Template Folder Index page. This is much less confusing than before, but still provides the information I wanted to have available on the html site (even if seldomly used). I'm sure this can be further improved from the current state, but this is now in a releasable form.